packages/plugin-essentials/sources/commands/install.ts

```diff
@@ -191,7 +192,7 @@ export default class YarnCommand extends BaseCommand {
     // lockfile - for example the PnP artifacts will also be locked.
     if (typeof this.frozenLockfile !== `undefined`) {
       const exitCode = await reportDeprecation(`The --frozen-lockfile option is deprecated; use --immutable and/or --immutable-cache instead`, {
-        error: !isGCF,
+        error: !isGCF && !isTravis,
```

Comment:
`ci-info`, which we use, has a `TRAVIS` constant we could use, same for `NETLIFY`

Reply:
Indeed, I'll update my Pr to use `ci-info` instead

Reply:
![image](https://user-images.githubusercontent.com/5300235/89933829-f28a6e00-dc0f-11ea-8dfb-95011f6b7989.png)
It seems that the library also support Netlify. Do you ant me to use it for Netlify too?

Reply:
Fix done, I now use the TRAVIS value of `ci-info`

Reply:
> It seems that the library also support Netlify. Do you ant me to use it for Netlify too?

The released version is checking a different variable than the one their master branch (and our code) checks, so unsure if it would work